src/app/services/github-service/github-service.service.ts

```diff
@@ -166,4 +169,16 @@ export class GithubServiceService {
     const url = environment.githubApiUrl +"/repos/"+projectLink+"/releases";
     return this.httpClient.get(url);
   }
+
+  getCompletedWorkflowRuns(projectLink: string) {
+    const url = environment.githubApiUrl +"/repos/"+projectLink+"/actions/runs?status=completed";
+    return this.httpClient.get(url);
+  }
+
+
+  getCompletedRuns(owner: string, repo: string): Observable<GitDetails[]> {
+    const url = `${environment.githubApiUrl}/repos/${owner}/${repo}/actions/runs?status=completed`;
```

Comment:
The url formula format can be simplyfied

Reply:
please update your changes